testing/simapp/simd/cmd/testnet.go

```diff
@@ -385,12 +382,12 @@ func writeFile(name string, dir string, contents []byte) error {
 	writePath := filepath.Join(dir)
 	file := filepath.Join(writePath, name)
 
-	err := tmos.EnsureDir(writePath, 0755)
+	err := tmos.EnsureDir(writePath, 0o755)
 	if err != nil {
 		return err
 	}
 
-	err = tmos.WriteFile(file, contents, 0644)
+	err = tmos.WriteFile(file, contents, 0o644)
```

Comment:
ditto

Reply:
yeah, this can be explained here:

https://github.com/golang/go/issues/33363